internal/oci/runtime_oci.go

```diff
@@ -821,6 +837,10 @@ func (r *runtimeOCI) StopLoopForContainer(c *Container) {
 	ctx, span := log.StartSpan(ctx)
 	defer span.End()
 
+	// Allow for SIGINT to correctly interrupt the stop loop, especially
+	// when CRI-O is run directly in the foreground in the terminal.
+	ctx, stop := signal.NotifyContext(ctx, os.Interrupt)
```

Comment:
I assume you've hit this while debugging. I think it's in part incomplete because I would guess there are other pieces in cri-o we're not correctly catching SIGINT. I am fine with this for now, but maybe we should plumb all the CRI calls with this so we have better INT handling throughout?

Reply:
@haircommander, yes.

We need to plumb context correctly where it should have been passed. But this would be more work and out of scope for this change.

A work cut out for a future Pull Request.